Pipfile

```diff
@@ -49,6 +49,7 @@ Jinja2 = ">=3.1.2, <4.0.0"
 tenacity = ">=8.2.2, <9.0.0"
 h2o-wave = "0.26"
 tiktoken = "*"
+hf-transfer = "*"
```

Comment:
Can we specify a version?

Reply:
yes sorry I already pushed it in meantime